### ISSUE TITLE
Tutorials

### ISSUE THREAD

**Issue by MisterCruz** [etiennepinchon/magixjs#2]
Love the documentation but 1 or 2 tutorials would make it even better!

Love this project and I'm looking forward to using this!

Keep up the good work :fire:

**Comment by pingram3541** [comment]
Yes, where do we go after 'hello".  I saw on product hunt "You can now publish playground projects right into the editor".  Where is this "editor" like in your demo?

**Comment by etiennepinchon** [comment]
Haha @pingram3541, it is part of the plan don't worry ;)
But before deploying more of MagiX I wanted to make sure people would like the framework first and patch any problems before showing more.

**Comment by pingram3541** [comment]
@etiennepinchon Awesome!  Makes sense to test the waters...but I'm just so impatient, can't wait lol =)

I've been having a go w/ App.js and having a lot of fun.  Thx for the new toy!

**Comment by etiennepinchon** [comment]
Haha thanks dude!
You can still use the framework in the terminal app (I know it is not super friendly but then you can create multiple files and everything).
I am going to publish some tutorials very soon and more of magiX should come by the end of this week.. :)

**Comment by pingram3541** [comment]
Hate to hijack this thread further but have you see how EVE does beautiful code in a documentation syntax.  You write the code as you explain the purpose.  It's a breath of fresh air when you ponder how awesome this type of coding would be for teams.

http://witheve.com/

**Comment by etiennepinchon** [comment]
@pingram3541 then lets do the same dude! x)

**Comment by pingram3541** [comment]
@etiennepinchon Yes! (fist pump)  I'm glad to help in any way I can too.  I'm still getting up to speed w/ Nodejs and CoffeScript but my heart is definitely in it dude!
